gabriel-portfolio/portfolio/src/main/webapp/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+      <title>Gabriel's Portfolio</title>
+      <link rel="stylesheet" href="style.css">
+      <script src="script.js"></script>
+    </head>
```

Comment:
Closing tags should have the same indentations as the opening tags. If you are using some editor app, that app might be misconfigured for html indentation since I found all your closing tags are off by 2 spaces.

Reply:
I see. I fixed the indentation problem and that will be reflected on my next PR.